crates/allocator/Cargo.toml

```diff
@@ -1,25 +1,25 @@
 [package]
 name = "ink_allocator"
-version = "4.2.0"
+version.workspace = true
 authors = ["Parity Technologies <admin@parity.io>", "Robin Freyler <robin@parity.io>"]
```

Comment:
Just a suggestion for @cmichi and @ascjones : Should we inherit `authors` from the workspace too? It appears that it should consistently be `"Parity Technologies <admin@parity.io>"`

Reply:
See [discussion](https://github.com/paritytech/ink/pull/1835#discussion_r1264674106) above. @Robbepop is the original author of some of the crates so those ones will remain not inhherited, and the rest inherit the default.